src/audio_excl.rs

```diff
@@ -88,31 +83,24 @@ impl FileLister {
         file_to_move: PathBuf,
         new_dir: PathBuf,
     ) -> io::Result<()> {
-        println!(
-            "moving {} to {}",
-            file_to_move.to_str().unwrap(),
-            new_dir.to_str().unwrap()
-        );
-        let expected_sec = match self.security.get(&file_to_move) {
-            None => {
-                return Err(io::Error::new(
-                    io::ErrorKind::NotFound,
-                    "No such file found",
-                ))
-            }
-            Some(sec) => sec,
-        };
+        println!("moving {:?} to {:?}", file_to_move, new_dir);
```

Comment:
Print the debug form and avoid an unwrap / give a better representation if there's non-printing garbage in the filename.